packages/components/select/__tests__/select.test.ts

```diff
@@ -2944,4 +2944,71 @@ describe('Select', () => {
       vi.useRealTimers()
     })
   })
+
+  it('not options keep the selected label', async () => {
```

Comment:
```suggestion
  it('should keep the selected label after filtering options', async () => {
```
Consider rephrasing the test name to it should keep the selected label after filtering options for better semantic.